app.py

```diff
@@ -64,7 +64,11 @@ def create_routes(app):
     api.add_resource(Leases, "lease")
     api.add_resource(Tickets, "tickets")
     api.add_resource(Ticket, "tickets/<int:id>")
-    api.add_resource(Note, "tickets/<int:id>/notes")
+    api.add_resource(
+        Note,
+        "tickets/<int:ticket_id>/notes",
+        "tickets/<int:ticket_id>/notes/<int:note_id>",
+    )
```

Comment:
This should be the following:
```suggestion
    api.add_resource(Notes, "tickets/<int:ticket_id>/notes")
    api.add_resource(Note,   "tickets/<int:ticket_id>/notes/<int:id>")
```

Note class acts on a single Note resource.
Notes class acts when a single Note resource is not provided. Such as retrieving multiple notes, or creating a note.